src/controllers/cohort.js

```diff
@@ -1,12 +1,20 @@
 import { createCohort } from '../domain/cohort.js'
 import { sendDataResponse, sendMessageResponse } from '../utils/responses.js'
+import * as validation from '../utils/validationFunctions.js'
 
 export const create = async (req, res) => {
+  const { startDate, endDate } = req.body
+
   try {
-    const createdCohort = await createCohort()
+    const { parsedStartDate, parsedEndDate } = validation.dateValidation(
+      startDate,
+      endDate
+    )
+    const createdCohort = await createCohort(parsedStartDate, parsedEndDate)
 
     return sendDataResponse(res, 201, createdCohort)
   } catch (e) {
-    return sendMessageResponse(res, 500, 'Unable to create cohort')
+    console.error(e)
```

Comment:
It looks like this is still missing context. Adding some context to to the error printed in the console would be helpful when debugging any errors in the future.